src/apis/btc-core.ts

```diff
@@ -1,31 +1,56 @@
-import { BlockApi, Status, TxApi } from "@interlay/esplora-btc-api";
+import { BlockApi, Status, TxApi, ScripthashApi, Transaction } from "@interlay/esplora-btc-api";
 import { AxiosResponse } from "axios";
+import * as bitcoinjs from "bitcoinjs-lib";
 
 const mainnetApiBasePath = "https://blockstream.info/api";
-const testnetApiBasePath = "https://blockstream.info/testnet/api";
+const testnetApiBasePath = "https://electr-testnet.do.polkabtc.io";
```

Comment:
Why are we declaring this here actually? We should use the definition directly from https://github.com/interlay/esplora-btc-api/blob/master/api.yaml#L8

Reply:
Good point. Then I'll change the testnet server URL to our node in the esplora wrapper

Reply:
I've had a look over docs and tutorials, and the `servers` object in OpenAPI only serves for showing a drop-down of available servers on swagger. It doesn't seem to be accessible once the API code is generated.
https://idratherbewriting.com/learnapidoc/pubapis_openapi_step3_servers_object.html

Nonetheless, I'll update esplora-btc-api anyways

Reply:
Ah, ok. Thanks, I wasn't aware of that. Then we'll keep it as is!